.github/workflows/push-image.yaml

```diff
@@ -0,0 +1,73 @@
+---
+name: Build and push image to ECR
+
+on:
+  workflow_dispatch:
+    inputs:
+      docker_base_distribution:
+        description: "Enter the Docker Base Distribution (bionic, focal, stretch, buster, or bullseye)"
+        required: true
+        type: string
+        default: bullseye
+      network_name:
+        description: "Enter the Network Name (berkeley or mainnet)"
+        required: true
+        type: string
+        default: mainnet
+      docker_base_tag_prefix:
+        description: "Enter the Base Tag Prefix: See https://github.com/MinaProtocol/mina/discussions/13056 for more details"
```

Comment:
Is this a correct link? I cannot figured out based on info there which Base Tag should be put here :sweat_smile:

Reply:
Not correct at all, changing